app/core/interfaces/numeric/index.js

```diff
@@ -68,31 +57,32 @@ define([
       'SMALLINT',
       'MEDIUMINT',
       'INT',
-      'NUMERIC',
       'FLOAT',
       'YEAR',
+      'DOUBLE',
       'VARCHAR',
       'CHAR',
-      'DOUBLE',
       'BIGINT'
     ],
     variables: [
       {id: 'size', type: 'String', default_value: 'large', ui: 'select', options: {options: {'large':__t('size_large'),'medium':__t('size_medium'),'small':__t('size_small')} }},
-      {id: 'placeholder_text', type: 'String', default_value: '', ui: 'textinput', char_length:200},
-      {id: 'allow_null', type: 'Boolean', default_value: false, ui: 'checkbox'},
```

Comment:
`allow_null` wasn't checked in any way and I thought the check would be the same as the one for `is_required`, please let me know if I overlooked something